src/js/components/widget.jsx

```diff
@@ -105,39 +109,46 @@ export class WidgetComponent extends Component {
             `sk-link-color-${isLinkColorDark ? 'dark' : 'light'}`
         ];
 
-        return (
-            <div id='sk-container'
-                 className={ classNames.join(' ') }
-                 onTouchStart={ this.onTouchStart }
-                 onClick={ this.onClick }
-                 onWheel={ this.onWheel }>
-                <MessageIndicator />
-                <div id='sk-wrapper'
-                     className={ wrapperClassNames.join(' ') }>
-                    <Header />
-                    <ReactCSSTransitionGroup component='div'
-                                             className='sk-notification-container'
-                                             transitionName='sk-notification'
-                                             transitionAppear={ true }
-                                             transitionAppearTimeout={ 500 }
-                                             transitionEnterTimeout={ 500 }
-                                             transitionLeaveTimeout={ 500 }>
-                        { notification }
-                    </ReactCSSTransitionGroup>
-                    <ReactCSSTransitionGroup component='div'
-                                             transitionName='settings'
-                                             transitionAppear={ true }
-                                             transitionAppearTimeout={ 250 }
-                                             transitionEnterTimeout={ 250 }
-                                             transitionLeaveTimeout={ 250 }>
-                        { settingsComponent }
-                    </ReactCSSTransitionGroup>
-                    { channelsComponent }
-                    <Conversation />
-                    { footer }
-                </div>
-            </div>
-            );
+        let badge;
+
+        if (displayStyle === DISPLAY_STYLE.BADGE && !appState.embedded) {
+            badge = <Badge shown={ !appState.widgetOpened } />;
```

Comment:
Why is it passing a `shown` prop instead of setting `badge` to `null` if widgetOpened is true?